### ISSUE TITLE
Something went wrong: Not enough funds.

### ISSUE THREAD

**Issue by rithvikvibhu** [kyokan/bob-wallet#267]
## Expected Behavior

Spending anywhere should work as expected.

## Current Behavior

Anything that requires spending gives this error in red. In this case, selecting "Send Max" in the Spend tab:
```
Something went wrong: Not enough funds. (available=0.0, required=4246.494314)
```
Now the required amount is correctly picked up as the same balance shown in the portfolio, but available is 0.0 for some reason. 

I claimed the airdrop coins (4246) an hour ago and the Portfolio shows the correct balance.
Is this because the transaction hasn't received enough confirmations? If so, it'd be nice if the unconfirmed amount is shown separately.

## Steps to Reproduce

1. Create a new wallet
2. Claim with hs-airdrop
3. After the balance is updated in portfolio, try to "Send Max" in the Send tab.

## Context (Environment)
Linux, built an AppImage.

## Logs
Will do, if needed.

## Screencap

Portfolio:
![image](https://user-images.githubusercontent.com/5113343/97987458-58e2ef00-1e01-11eb-9cb3-c4b35d42eb51.png)

Send screen (same error when trying to start an auction):
![image](https://user-images.githubusercontent.com/5113343/97987722-c7c04800-1e01-11eb-9938-da6b070b6341.png)

**Comment by pinheadmz** [comment]
Airdrop coins require 100 confirmations before they are spendable. That is a network consensus rule and applies to all outputs of all coinbase transactions. Try again in one day ;-)

**Comment by pinheadmz** [comment]
Airdrop coins require 100 confirmations before they are spendable. That is a network consensus rule and applies to all outputs of all coinbase transactions. Try again in one day ;-)

**Comment by rithvikvibhu** [comment]
Ah, thanks for that! I'll wait, no problem.
A suggestion, it'd be nice if the app mentioned this somewhere (next to airdrop txs), for people new to Handshake and a more specific error message :)

**Comment by pinheadmz** [comment]
Agreed - I think you should re-open the issue for that feature request
